src/Assert/Assert.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Assert;
+
+use Wikimedia\Assert\Assert as WikimediaAssert;
+use Wikimedia\Assert\ParameterAssertionException;
+
+/**
+ * Provides functions to assure values meet certain preconditions relevant
+ * in Wikibase.
+ *
+ * @see Wikimedia\Assert\Assert
+ *
+ * @since 3.7
+ *
+ * @license GPL-2.0+
+ */
+class Assert {
+
+	/**
+	 * @param string $value
+	 * @param string $name
+	 *
+	 * @throws ParameterAssertionException if $value is not an allowable repository name.
+	 */
+	public static function parameterIsAllowableRepositoryName( $value, $name ) {
```

Comment:
"valid" is ok here. If you want to follow the example of XML, you could used "well formed" as the predicate.